main.js

```diff
@@ -5,10 +5,25 @@ const url = require('url')
 const isDev = require('electron-is-dev')
 const path = require('path')
 
+// use this to redirect a request path to a url
+const REDIRECTIONS = {
+  // request path: resource path
+  'tree-sitter-bash.wasm': getURL('dist/tree-sitter-bash.wasm') // a dependency for curlConverter module is using absolute path for this
+}
```

Comment:
This can be used to fix absolute path problems inside a dependency.

Simply add the part after file:/// and the root as key and provide URL path for real resource

Reply:
E.g.: file:///C:/tree-sitter-bash.wasm -> tree-sitter-bash.wasm for the key can be used